tests/e2e/utils/requestHandlers/CheApiRequestHandler.ts

```diff
@@ -0,0 +1,43 @@
+/*********************************************************************
+ * Copyright (c) 2019 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import axios, { AxiosResponse } from 'axios';
+import { TestConstants } from '../../TestConstants';
+import { TYPES } from '../../inversify.types';
+import { inject, injectable } from 'inversify';
+import { IHeaderHandler } from './IHeaderHandler';
+
+@injectable()
+ export class CheApiRequestHandler {
+
+    constructor(@inject(TYPES.HeaderHandler) private readonly headerHandler: IHeaderHandler) {
+    }
+
+    async get(url: string) : Promise<AxiosResponse> {
```

Comment:
On my opinion the ```relativeUrl``` will be more convenient here.
Because simple ```url``` may leads to confusion,
for example if I see ```url``` I think about full url ```http://www.example....```
but in this case, uses relative URL.